packages/react-wildcat/src/server.js

```diff
@@ -107,6 +107,21 @@ function start() {
                 }));
             });
 
+            // Allow custom middleware priority over react/jspm/render from below
+            // otherwise server-only routes will receive 404's.
+            if (appServerSettings.middleware && appServerSettings.middleware.length) {
+                appServerSettings.middleware.forEach(function eachCustomAppMiddleware(middlewareConfigFunction, index) {
+                    if (typeof middlewareConfigFunction === "function") {
+                        middlewareConfigFunction(app, wildcatConfig);
+                    } else {
+                        var errorMsg = "Middleware at serverSettings.appServer.middleware[" + index + "] could not be correclty initialize. Expecting middleware to have the follow the following signature:\n" +
+                        "    function(app, wildcatConfig) { /* custom middleware */ }\n" +
+                        "    Erroring middleware: " + (middlewareConfigFunction || '').toString();
+                        logger.error(errorMsg, middlewareConfigFunction);
+                    }
```

Comment:
Maybe use template strings here to make this more readable? Also: "initialize" -> "initialized".